src/libraries/Microsoft.Extensions.Http/src/DependencyInjection/HttpClientFactoryServiceCollectionExtensions.cs

```diff
@@ -450,7 +450,7 @@ public static IHttpClientBuilder AddHttpClient(this IServiceCollection services,
 
             AddHttpClient(services);
 
-            string name = TypeNameHelper.GetTypeDisplayName(typeof(TClient), fullName: false);
+            string name = TypeNameHelper.GetTypeDisplayName(typeof(TImplementation), fullName: false);
```

Comment:
This makes the documentation on the method (and any that call it) now incorrect. (line 440)

Reply:
You're right I'm gonna change the documentation